ig.ini

```diff
@@ -0,0 +1,12 @@
+[IG]
+; ***********************************************************************************************
+; *                               WARNING: DO NOT EDIT THIS FILE                                *
+; *                                                                                             *
+; * This file is generated by SUSHI. Any edits you make to this file will be overwritten.       *
+; *                                                                                             *
+; * This ig.ini was generated using the template property in config.yaml. To provide your own   *
+; * ig.ini, create an ig.ini file in the ig-data folder with required properties: ig, template. *
+; * See: https://build.fhir.org/ig/FHIR/ig-guidance/using-templates.html#root                   *
+; ***********************************************************************************************
+ig = input/ImplementationGuide-saraalert-fhir.json
+template = fhir.base.template#current
```

Comment:
This line should change to 
```
ig = fsh-generated/resources/ImplementationGuide-saraalert-fhir.json
```
once `input/ImplementationGuide-saraalert-fhir.json` is removed, as mentioned in the comment below.

Reply:
https://github.com/SaraAlert/saraalert-fhir-ig/pull/6/commits/e90a101d4dc0c7a2e8219aacd6e9bf7cf6da4d62